### PR TITLE
Updating tests

### PR DESCRIPTION
**Pull request by elsavira**
Updated CI tests and module workspace folder.

### CODE REVIEW

File: src/some-controller.hpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#ifndef SOME_CONTROLLER_HPP
+#define SOME_CONTROLLER_HPP
+
+#include "wrap-hwlib.hpp"
+
+class SomeController {
```
Comment (NielsdeWaal) [inline src/some-controller.hpp]: This class is missing is the doxygen documentation.

File: src/some-controller.hpp [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                c |= in.get();
+            }
+
+            *buffer = c;
+
+            if (c == 0) {
+                break;
+            }
+
+            buffer++;
```
Comment (NielsdeWaal) [inline src/some-controller.hpp]: This usage of the pointer to buffer seems dangerous. There is no documentation to support these choices.

**Review by NielsdeWaal — CHANGES_REQUESTED** [review]
Please fix the highlighted issues.

File: src/some-controller.hpp (hunk 2 above)
Comment (elsavira) [reply src/some-controller.hpp]: These files were added in the recent Arduino template. Will remove.

**Comment by elsavira** [comment]
Requested changes were implemented. File removed.

**Review by NielsdeWaal — APPROVED** [review]

**Pull request merged**